public/app/features/alerting/unified/components/rule-editor/GrafanaEvaluationBehavior.tsx

```diff
@@ -101,6 +124,14 @@ export const GrafanaEvaluationBehavior: FC = () => {
           </Field>
         </div>
       </Field>
+      {exceedsGlobalEvaluationLimit && (
+        <Alert severity="warning" title="Global evalutation interval limit exceeded">
+          A minimum evaluation interval of{' '}
+          <span className={styles.globalLimitValue}>{config.unifiedAlerting.minInterval}</span> have been configured in
+          Grafana and will be used for this alert rule. <br />
```

Comment:
```suggestion
          Grafana. <br />
```

I think this is OK as this warning is shown when configuring the `For` interval.